pantos/validatornode/blockchains/ethereum.py

```diff
@@ -31,7 +31,7 @@
 from pantos.validatornode.entities import CrossChainTransfer
 
 _HUB_TRANSFER_TO_FUNCTION_SELECTOR = '0x92557c8a'
-_HUB_TRANSFER_TO_GAS = 250000
+_HUB_TRANSFER_TO_GAS = 400000
```

Comment:
The amount of gas required for a `transferTo` is variable and depends on the number of validators. The more validators we have, the more expensive it is (because we check the signatures iteratively on-chain). 

Do you think it's worth to define this value based on the amount of validators configured? (i.e. we start with a minimum amount that we assume for 0 secondary validators, and for each extra secondary validator configured, we add some amount of gas -> ideally we should run some tests here to define a proper value). Obviously it would take some effort to define accurate values for the base amount and added amount, so I'm just wondering if it's worth or not.

Reply:
The fixed gas amount specified would have been sufficient for our mainnet setup. But I agree, it's better to define it based on the number of signers of a transfer. I've changed this now. The gas amount now defined is rather generous to avoid insufficient gas problems.